README.md

```diff
@@ -13,65 +16,91 @@ SQL recipes for Data Export analysis are in the [Data Export Cookbook](https://g
-        - Linux & Mac: `$HOME/go/bin/hauser -c <your updated config file>`
-        - Windows: `%USERPROFILE%\go\bin\hauser -c <your updated config file>`
-    * If you _DID_ set a custom `GOPATH`...
-        - Linux & Mac: `$GOPATH/bin/hauser -c <your updated config file>`
-        - Windows: `$GOPATH\bin\hauser -c <your updated config file>`
-    
-## How It Works
-When first run, `hauser` will query FullStory's [data export API](http://help.fullstory.com/develop-rest) to find the earliest export file available. `hauser` will then download all available export files, performing some light transformation for [custom user vars](http://help.fullstory.com/develop-js/setuservars?from_search=17717406) before loading it into the warehouse.
+1. Download the latest [release binary](https://github.com/fullstorydev/hauser/releases)
+2. Download the included `example-config.toml` file and customize it for your environment,
+   including your FullStory API key, warehouse host, and credentials. AWS credentials (for S3) come from your local environment.
+3. Assuming the binary and updated config are in the current directory, run:
+```bash
+./hauser -c myconfig.toml
+```
 
-`hauser` will work through all available export files serially. When no further export files are available, `hauser` will sleep until there is a new one available, which will be processed immediately.
+### Important Configuration Fields
 
-Export files may be processed one at a time, or they may be grouped into batches by day using the boolean config option `GroupFilesByDay`.  When grouping is enabled, export files are still processed serially, but all files having the same date (in UTC) will be combined into a single file before upload to the target warehouse.  Grouping files is helpful for loading large amounts of historical data, when the total number of load operations might reach some quota.  BigQuery, for example, limits the number of loads per day on a single table to 1000.
+#### `ExportDuration`
+Determines the time range for each export which ultimately determines the size of each exported file.
+The size of this file will be based on the amount of traffic that your FullStory account records within the
+specified duration.
+The default is 1 hour, but if a different file size is desirable, this can be modified to meet your specific needs. The
+max duration is 24 hours.
 
-`hauser` can safely be stopped and restarted. For Redshift and BigQuery, it uses the `SyncTable` to keep track of what export files have been processed, and will restart from the last known sync point.
+#### `ExportDelay`
+Determines how long to wait before creating an export.
+This delay is necessary because there is some latency between when an event is recorded and when it is available and complete.
+24 hours is the default, but is also fairly conservative. In many cases this can be reduced safely to 3 hours, but note that
+events from "[swan songs]" may not be available.
```

Comment:
`swan songs` can it link to the reference below?

Reply:
Yes. You could check by viewing the rich diff, but it's also part of the spec and is called a [shortcut reference link](https://github.github.com/gfm/#shortcut-reference-link)